ara/templates/playbook.html

```diff
@@ -93,8 +106,9 @@
     <tr>
       <td>{{ task.offset_from_playbook |timefmt }}</td>
       <td>{{ task.duration |timefmt }}</td>
+      <td>{{ task.name }}</td>
       <td>{{ task.action }}</td>
-      <td>{{ task.path|pathtruncate }}</td>
+      <td>{{ task.file.path|pathtruncate }}</td>
       <td>{{ task.lineno }}</td>
```

Comment:
I have no clue how to do that but I could definitely see this line link to the file contents page with the line highlighted (similar to github)